assets/samplesheet.csv

```diff
@@ -1,3 +1,7 @@
 sample,cram,crai,bed
 1234N,https://github.com/nf-core/test-datasets/raw/sarek/testdata/recalcram/1234N.recal.cram,https://github.com/nf-core/test-datasets/raw/sarek/testdata/recalcram/1234N.recal.cram.crai,https://github.com/nf-core/test-datasets/raw/sarek/testdata/target.bed
 9876T,https://github.com/nf-core/test-datasets/raw/sarek/testdata/recalcram/9876T.recal.cram,https://github.com/nf-core/test-datasets/raw/sarek/testdata/recalcram/9876T.recal.cram.crai,https://github.com/nf-core/test-datasets/raw/sarek/testdata/target.bed
```

Comment:
```suggestion
```

These can be removed since they were not that useful for testing

Reply:
Want me to add negative controls as well? Or are the four positive ones enough for now?